### ISSUE TITLE
does no show all the items in the list

### ISSUE THREAD

**Issue by olivoil** [TanStack/virtual#356]
### Describe the bug

When displaying a large number of items, scrolling at the bottom of the scroll element does not reach the end of the list

### Your minimal, reproducible example

https://codesandbox.io/s/amazing-saha-f0dqsh

### Steps to reproduce

1. Go to the default fixed example at https://tanstack.com/virtual/v3/docs/examples/react/fixed
2. Edit `scr/main.tsx`, and change the `amount` virtualizer option from 10,000 to 1,000,000
3.  Scroll to the bottom of the list view in the browser
4. Notice the last visible item index is #958,697

### Expected behavior

The last index should be 999,999, not 958,697. Not all items are being displayed in the list.

### How often does this bug happen?

Every time

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/264142/182006941-ab976f92-343b-4e4c-aa0d-1b7848ad27c3.png)

### Platform

tested on windows 11, on edge Version 103.0.1264.71 (Official build) (64-bit)

### tanstack-virtual version

3.0.0-beta.14

### Terms & Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I understand that if my bug cannot be reliable reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by ardeora** [comment]
Hello! I think this is not a tanstack-virtual issue. There is more explaination on react-window/virtualized's discussion linked here
https://github.com/bvaughn/react-window/issues/108
https://github.com/bvaughn/react-virtualized/issues/193#issuecomment-211467531 

Try making the size of each row from 35px -> 10px and you will see all 1 million rows :D 

TLDR: It's a browser limitation.

**Comment by tannerlinsley** [comment]
Indeed. This is a limitation on max element size.
